src/risedevtool/src/task/frontend_service.rs

```diff
@@ -97,6 +98,13 @@ impl Task for FrontendService {
             ctx.pb.set_message("started");
         } else {
             ctx.pb.set_message("user managed");
+            writeln!(
```

Comment:
I remember we have printed this in RiseDev's log?

Reply:
Only for compute

Reply:
Good catch. 😄